src/babel/index.ts

```diff
@@ -131,6 +136,11 @@ export default function ({
             const importSpecifierStringLiteral = t.stringLiteral(
               importSpecifier
             );
+            const processPath = process.cwd();
```

Comment:
Have we verified that this works correctly also in a case like Jira SSR, where webpack is called from a different subfolder?
My gut tells me that we should be looking for the concept of a root folder. But not sure we can find it without some user input/config

Reply:
Have now fixed this